package.json

```diff
@@ -41,21 +36,21 @@
     "end-to-end",
     "e2e"
   ],
-  "author": "Kent C. Dodds <me@kentcdodds.com> (https://kentcdodds.com)",
+  "author": "Kent C. Dodds <kentcdodds@gmail.com> (https://kentcdodds.com)",
```

Comment:
I'd prefer this to be: me@kentcdodds.com

Reply:
I just copied it over from `generator-kcd-oss`, will change it again 🙂